README.md

```diff
@@ -1,6 +1,13 @@
-# Jellyfish
+<div style="text-align: center">
 
-A self-hosted screenshots/files hosting system.
+![](img/Jellyfish_light.svg#gh-light-mode-only)
+![](img/Jellyfish_dark.svg#gh-dark-mode-only)
+
+</div>
+
+<p align="center">
+<i>A self-hosted screenshots/files hosting system.</i>
+</p>
```

Comment:
Put everything in the same div, for semantic reasons, and because it kind of helps the crawler, add an alt to the image (text between the square brackets), and make the subtitle an h2 (again, semantics)

```html
<div align="center">

![Jellyfish logo](img/Jellyfish_light.svg#gh-light-mode-only)
![Jellyfish logo](img/Jellyfish_dark.svg#gh-dark-mode-only)

<h2><i>A self-hosted screenshots/files hosting system.</i></h2>

</div>
```

Reply:
I find `h2` a bit too big.. Is it okay to use `h3` instead ?

Reply:
This is sad, but i can't find a way to make in place style work with it to change the font size ; if you can, please do so, if not, then h3 we go

Reply:
I think it's not worth the effort, let's go for `h3`!